types_test.go

```diff
@@ -628,6 +628,84 @@ func TestIsHostname(t *testing.T) {
 	}
 }
 
+func TestIsGCPProjectId(t *testing.T) {
+	tests := []struct {
+		name   string
+		target string
+		want   bool
+	}{
+		{
+			name:   "GCP Project ID with length equal to or greater than 6 characters",
+			target: "rabbit",
+			want:   true,
+		},
+		{
+			name:   "GCP Project ID length should not less than 6 characters",
```

Comment:
`s/GCP Project ID length should not less than 6 characters/GCP Project ID length should not be less than 6 characters` ?